{"name": "Логин курьера с некорректным логином", "status": "passed", "description": "Тест что курьер не может залогиниться успешно с неверным логином.", "steps": [{"name": "Создание курьера", "status": "passed", "parameters": [{"name": "courier", "value": "<models.courier.Courier object at 0x0000021743A13430>"}], "start": 1716464702148, "stop": 1716464702642}, {"name": "Логин курьера", "status": "passed", "parameters": [{"name": "credentials", "value": "<models.courier.Courier object at 0x0000021743A13430>"}], "start": 1716464702642, "stop": 1716464703091}, {"name": "Удаление курьера", "status": "passed", "steps": [{"name": "Логин курьера", "status": "passed", "parameters": [{"name": "credentials", "value": "<models.credentials.Credentials object at 0x0000021743A138E0>"}], "start": 1716464703091, "stop": 1716464703545}, {"name": "Удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "312762"}], "start": 1716464703545, "stop": 1716464703983}], "parameters": [{"name": "data", "value": "<models.credentials.Credentials object at 0x0000021743A138E0>"}, {"name": "client", "value": "<client.courier_client.CourierClient object at 0x0000021743A13550>"}], "start": 1716464703091, "stop": 1716464703983}], "start": 1716464702148, "stop": 1716464703983, "uuid": "f5ec8b49-559c-4cbd-b0f1-5256500f7063", "historyId": "86209548c51fbc8a3671e326c5bb0e3d", "testCaseId": "86209548c51fbc8a3671e326c5bb0e3d", "fullName": "tests.courier.test_courier_login.TestCourierLogin#test_courier_can_not_log_in_with_incorrect_login", "labels": [{"name": "parentSuite", "value": "tests.courier"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "10248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.courier.test_courier_login"}]}